Description: Ignore warnings from colord and Avahi in the tests, as they change the expected number of warnings.
Author: Martin Pitt <mpitt@debian.org>

Index: cups-1.5.2/test/run-stp-tests.sh
===================================================================
--- cups-1.5.2.orig/test/run-stp-tests.sh	2012-03-01 07:16:41.180825081 +0100
+++ cups-1.5.2/test/run-stp-tests.sh	2012-03-01 07:17:29.660827422 +0100
@@ -767,7 +767,7 @@
 fi
 
 # Warning log messages
-count=`$GREP '^W ' /tmp/cups-$user/log/error_log | wc -l | awk '{print $1}'`
+count=`$GREP '^W ' /tmp/cups-$user/log/error_log | grep -v 'org.freedesktop.ColorManager' | grep -v 'Avahi client failed: -26' | wc -l | awk '{print $1}'`
 if test $count != 9; then
 	echo "FAIL: $count warning messages, expected 9."
 	$GREP '^W ' /tmp/cups-$user/log/error_log
